README.md

```diff
@@ -5,13 +5,104 @@ Group a list of signals into one hash table signal. The output signal will conta
 
 If *one_value* is True, the Signal attributes will be just a single matching value instead of a list of all matching values. If multiple matches, then the last signal processed will be the value used.
 
+If *group_by* is used, a Signal attribute (key) with the default (pre-assigned) name *group* will have the value of *group_by* in the output signal.
+
+**Example:**
+
+_Signals Input_:
+
+```python
+[
+{ "type": "important", "color": "red", "age": 32 },
+{ "type": "minor", "color": "red", "age": 9 },
+{ "type": "important", "color": "orange", "age": 14 },
+{ "type": "standard", "color": "red", "age": 1 }
+]
+```
+
+_Block Config with key based on "type"_:
+
+```python
+key : {{ $type }}
+value : {{ $age }}
```

Comment:
These dollar signs are doing a weird highlight thing, even here: https://github.com/jennyknuth/hash_table/tree/nio2

Is there a way to fix that?